### PR TITLE
RF01TC045_TC_DanielFernandez-03

### PR DESCRIPTION
**Pull request by danielrpg**
UI >Asignacion de Equipamiento a Personal > Verificar formulario de asignacion Equipamiento a Personal

### CODE REVIEW

**Comment by danielrpg** [comment]
image

**Comment by danielrpg** [comment]
image

**Comment by marbusjim** [comment]
Please review your TC and titles

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java [hunk 1]
```diff
@@ -15,6 +16,9 @@
     @FindBy(id = "personal")
     private WebElement subMenuPersonnel;
 
+    @FindBy(xpath = "//button[@id='smAssign']")
+    private WebElement smAssignSubMenu;
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java]: use subMenuSmAssign

**Review by ctola91 — CHANGES_REQUESTED** [review]

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java (hunk 1 above)
Comment (marbusjim) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java]: change to subMenuSmAssign

**Review by marbusjim — COMMENTED** [review]
please solve the comments

**Review by marbusjim — APPROVED** [review]
Approve

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java [hunk 2]
```diff
@@ -15,6 +16,9 @@
     @FindBy(id = "personal")
     private WebElement subMenuPersonnel;
 
+    @FindBy(xpath = "//button[@id='smAssign']")
```
Comment (ivanMise) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonal.java]: use id please

**Review by ivanMise — APPROVED** [review]

**Review by ctola91 — APPROVED** [review]

**Pull request merged**